pymc/variational/opvi.py

```diff
@@ -362,9 +363,9 @@ def step_function(
             total_grad_norm_constraint=total_grad_norm_constraint,
         )
         if score:
-            step_fn = aesara.function([], updates.loss, updates=updates, **fn_kwargs)
+            step_fn = compile_pymc([], updates.loss, updates=updates, **fn_kwargs)
         else:
-            step_fn = aesara.function([], None, updates=updates, **fn_kwargs)
+            step_fn = compile_pymc([], None, updates=updates, **fn_kwargs)
```

Comment:
My changes broke this line. What is the goal of this `step_fn`, to just change the `updates`?

Reply:
Updated it, should be okay now